fly/commands/internal/setpipelinehelpers/atc_config.go

```diff
@@ -28,7 +28,7 @@ type ATCConfig struct {
 	SkipInteraction  bool
 	CheckCredentials bool
 	DryRun           bool
-	CommandWarnings  []concourse.ConfigWarning
+	CommandErrors    []atc.ConfigErrors
```

Comment:
I don't think this field is will be needed anymore if you consolidate to just return `error` instead of using `[]atc.ConfigErrors` everywhere still.